src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets.properties

```diff
@@ -52,6 +52,7 @@ Please file a bug against the javadoc tool via the Java bug reporting page\n\
 (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com)\n\
 for duplicates. Include error messages and the following diagnostic in your report. Thank you.
 doclet.File_not_found=File not found: {0}
+doclet.snippet_file_not_found=file not found on source path or snippet path: {0}
```

Comment:
Is this error message expected to start with a lower case letter? Other error messages in this file seem to start with an upper case letter.

Reply:
It was deliberate that it didn't. The error messages are inconsistent, and generally phrases should not be capitalized. Note that many of the surrounding resources are labels to appear in the generated docs, and so have different rules/guidelines for capitalization. (This is another reason for the long overdue cleanup of resource names and values).